helpers/compile/build.ts

```diff
@@ -184,7 +184,8 @@ const watch =
         console.error(rebuildResult.message)
       }
 
-      console.log(`${Date.now() - timeBefore}ms [${options.name ?? ''}]`)
+      const timeAfter = Math.round(performance.now())
+      console.log(`${timeAfter - timeBefore}ms [${options.name ?? ''}]`)
```

Comment:
A bit late here, but I think rounding should happen on the final expression (i.e. `Math.round(timeAfter - timeBefore)`) and not on each term so that we don't accumulate rounding errors.

It's nothing critical but would be nice to do, I can open a PR.

Reply:
Feel free to open a PR. My current thinking is that we don't need to be precise in all the places I changed.